mycodo/mycodo_flask/templates/pages/function_options/trigger_options/run_pwm_method.html

```diff
@@ -3,7 +3,7 @@
   <div>
     <select class="form-control form-tooltip form-dropdown" id="unique_id_1" name="unique_id_1" data-placement="top" title="{{_('Run this duration method, using the output value to set the Duty Cycle of a PWM output')}}">
       <option value="">{{dict_translation['select_one']['title']}}</option>
-      {%- for each_method in method if each_method.method_type == 'Duration' -%}
+      {%- for each_method in method -%}
```

Comment:
Search for references and testing has shown, that this was an unnecessary restriction, at the other method types perfectly work with run_pwm_method